joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -485,7 +489,7 @@ CallbackReturn JointTrajectoryController::on_configure(const rclcpp_lifecycle::S
       // TODO(anyone): This flag is not used for now
       // There should be PID-approach used as in ROS1:
       // https://github.com/ros-controls/ros_controllers/blob/noetic-devel/joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h#L283
-      use_closed_loop_pid_adapter = true;
+      use_closed_loop_pid_adapter_ = true;
```

Comment:
Can we rename a variable in a separate PR? It would be a bit cleaner and easier to back port.